src/ResourceManager/Insights/Commands.Insights/Alerts/RemoveAzureRmAlertRuleCommand.cs

```diff
@@ -49,8 +50,8 @@ public class RemoveAzureRmAlertRuleCommand : ManagementCmdletBase
         /// </summary>
         protected override void ProcessRecordInternal()
         {
-            AzureOperationResponse result = this.InsightsManagementClient.AlertOperations.DeleteRuleAsync(resourceGroupName: this.ResourceGroup, ruleName: this.Name).Result;
-            WriteObject(result);
+            this.InsightsManagementClient.AlertRules.DeleteAsync(resourceGroupName: this.ResourceGroup, ruleName: this.Name).Wait();
+            WriteObject("");
```

Comment:
You should not use ```WriteObject ("")``` - this pushes a string onto the output stream.  The policy is that you must signal breaking changes to users a release or more in advance using a deprecation warning.